[2023-09-11T04:56:16.384+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test01.test02 manual__2023-09-11T04:56:07.702600+00:00 [queued]>
[2023-09-11T04:56:16.389+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test01.test02 manual__2023-09-11T04:56:07.702600+00:00 [queued]>
[2023-09-11T04:56:16.390+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-11T04:56:16.397+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): test02> on 2023-09-11 04:56:07.702600+00:00
[2023-09-11T04:56:16.401+0000] {standard_task_runner.py:57} INFO - Started process 66 to run task
[2023-09-11T04:56:16.404+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'test01', 'test02', 'manual__2023-09-11T04:56:07.702600+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/Orders_realPaidPrice.py', '--cfg-path', '/tmp/tmp5pgezkaf']
[2023-09-11T04:56:16.405+0000] {standard_task_runner.py:85} INFO - Job 61: Subtask test02
[2023-09-11T04:56:16.441+0000] {task_command.py:415} INFO - Running <TaskInstance: test01.test02 manual__2023-09-11T04:56:07.702600+00:00 [running]> on host 4b74fbf5f024
[2023-09-11T04:56:16.495+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test01' AIRFLOW_CTX_TASK_ID='test02' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T04:56:07.702600+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-11T04:56:07.702600+00:00'
[2023-09-11T04:56:16.717+0000] {logging_mixin.py:151} INFO - s3 bucket connected!
[2023-09-11T04:56:16.968+0000] {logging_mixin.py:151} INFO - 데이터가 S3 버킷 wingeatdata의 practice/2023-9-8-5.csv 위치에 저장되었습니다.
[2023-09-11T04:56:16.972+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-11T04:56:16.989+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test01, task_id=test02, execution_date=20230911T045607, start_date=20230911T045616, end_date=20230911T045616
[2023-09-11T04:56:17.051+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-11T04:56:17.067+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
